src/modelbench/run.py

```diff
@@ -130,45 +139,61 @@ def find_suts_for_sut_argument(sut_args: List[str]):
     return suts
 
 
-def score_benchmarks(benchmarks, suts, max_instances, debug, parallel=True):
+def score_benchmarks(benchmarks, suts, max_instances, machine_reports=False, parallel=True):
     secrets = load_secrets_from_config()
+
+    # Count total number of tests * SUTs to run.
+    total = 0
+    for b in benchmarks:
+        for h in b.hazards():
+            total += len(h.tests(secrets=secrets))
+    total *= len(suts)
+
     if parallel:
-        f = functools.partial(score_a_sut, benchmarks, max_instances, secrets, debug)
-        with Pool(len(suts)) as p:
-            results = p.map(f, suts)
+        with Manager() as manager:
+            shared_count = manager.Value(ctypes.c_double, 0.0)
+            lock = manager.Lock()
+            progress = ProgressTracker(total, machine_reports, shared_count, lock)
+            f = functools.partial(score_a_sut, benchmarks, max_instances, secrets, progress)
+            with Pool(len(suts)) as p:
+                results = p.map(f, suts)
+                p.close()
+                p.join()
             return list(itertools.chain(*results))
     else:
+        progress = ProgressTracker(total, machine_reports)
         benchmark_scores = []
         for sut in suts:
-            sut_scores = score_a_sut(benchmarks, max_instances, secrets, debug, sut)
+            sut_scores = score_a_sut(benchmarks, max_instances, secrets, progress, sut)
             benchmark_scores.extend(sut_scores)
         return benchmark_scores
 
 
-def score_a_sut(benchmarks, max_instances, secrets, debug, sut):
+def score_a_sut(benchmarks, max_instances, secrets, progress, sut):
     sut_scores = []
-    echo(termcolor.colored(f'Examining system "{sut.display_name}"', "green"))
+    logging.info(termcolor.colored(f'Examining system "{sut.display_name}"', "green"))
```

Comment:
Instead of hardcoding the `"green"` literal here and elsewhere, I'd wrap the `termcolor.colored("message", "color")` in a function like 

```python

INFO_COLOR = "green"

...

def info_message(message): 
    return termcolor.colored(message, INFO_COLOR)
```

Reply:
Yeah, I actually thought about configuring the logger to handle the colors. But I think these kinds of updates to the regular logging might be a bit out of the scope of this PR.